lib/global_config.js

```diff
@@ -11,6 +11,7 @@ const Errors = require('./errors');
 const Logger = require('./logger');
 
 let insecureConnect = false;
+let xmlParserConfiguration = null;
```

Comment:
should it be defined here or closer to the place where it is used?

Reply:
Could be closer ....